code/modules/antagonists/traitor/objectives/hack_comm_console.dm

```diff
@@ -44,6 +44,10 @@
 	return COMPONENT_CANCEL_ATTACK_CHAIN
 
 /datum/traitor_objective/hack_comm_console/proc/begin_hack(mob/user, obj/machinery/computer/communications/target)
+	if(!target.can_hack())
+		target.balloon_alert(user, "can't hack this console!")
```

Comment:
This feedback is quite non discript. There's nothing in game that suggests to the player as to why they can't hack the console, and no obvious in fiction reason.

Reply:
something along the lines of Signal too weak, or can't hack in maintenance etc.
Something to make it clear to the players who don't know about this feature WHY not, and bonus points if you can give explanation beyond it being an arbitrary game mechanic.

Reply:
"You can't send the signal from here! The signal is too weak in that area to connect to syndicate channels!"
Something like this.